lib/index.js

```diff
@@ -230,11 +230,12 @@ const completionItem = (item, shell) => {
  * Bash needs in addition to filter out the args for the completion to work
  * (zsh, fish don't need this).
  *
- * @param {Array.<CompletionItem | String>} args to log, Strings or Objects with name and
+ * @param {Array.<CompletionItem | String>} args - to log, Strings or Objects with name and
  * description property.
  * @param {SupportedShell} shell
+ * @param {(message: String) => void} log - Function to actually log to the console, usually `console.log`
  */
-const log = (args, shell) => {
+const logCompletion = (args, shell, log) => {
```

Comment:
I don't think a breaking change is needed in this case. The function can remain log. The new argument can be `logCompletion` and it can be optional. If not set, `console.log` is used.